app/views/mno/extra_mobile_data_requests/search_results.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :title, t('page_titles.requests_for_extra_mobile_data') %>
+<% content_for :before_content, govuk_link_to('Back', mno_extra_mobile_data_requests_path, class: 'govuk-back-link') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <%= pagy_info(@pagination, "requests").capitalize.html_safe %> found
```

Comment:
I think this does this:

![Screen Shot 2021-01-25 at 14 54 46](https://user-images.githubusercontent.com/319055/105722235-5df59d00-5f1d-11eb-9ed3-81f202add372.png)

Reply:
Good catch. Done